### PR TITLE
docs: french translation

### PR DESCRIPTION
**Pull request by Geminii**
Hi 👋 
There is several missing pages about the french documentation particularly for creation of docs themes 😀

## Types of changes
- [x] Documentation 
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

I added the missing pages and examples like : 
- integrations.md
- snippets.md
- theme/docs.md
- examples/tailwindcss-typography.md
- examples/docs-theme.md
- examples/basic.md

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

**Review by Atinux — APPROVED** [review]

File: docs/content/fr/examples/docs-theme.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Exemple de thèmes de documentation
```
Comment (Atinux) [inline docs/content/fr/examples/docs-theme.md:2]:
```suggestion
title: Exemple du thème de documentation
```

File: docs/content/fr/examples/tailwindcss-typography.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: Exemple TailwindCSS Typography
```
Comment (Atinux) [inline docs/content/fr/examples/tailwindcss-typography.md:2]:
```suggestion
title: Exemple avec TailwindCSS Typography
```

File: docs/content/fr/examples/tailwindcss-typography.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+menuTitle: TailwindCSS Typography
+description: 'Exemple en direct de contenu Nuxt avec le plugin TailwindCSS Typography sur CodeSandbox.'
```
Comment (Atinux) [inline docs/content/fr/examples/tailwindcss-typography.md:4]:
```suggestion
description: 'Exemple de contenu Markdown avec Nuxt et le plugin TailwindCSS Typography sur CodeSandbox.'
```

**Review by benjamincanac — APPROVED** [review]

**Comment by benjamincanac** [comment]
@Geminii Thank you for this PR, it was so much needed 🙂

**Pull request merged**